oneanddone/tests/functional/conftest.py

```diff
@@ -71,3 +76,10 @@ def task(base_url, is_local):
     if is_local:
         from oneanddone.tasks.tests import TaskFactory
         return TaskFactory.create()
+
+
+@pytest.fixture(scope='session')
```

Comment:
This looks like it might be an example of creating such a task: https://github.com/mozilla/oneanddone/blob/7195406ba41fa8e2178b06a9ee1c475f3322749e/oneanddone/tasks/tests/test_api.py#L64-L97